### PR TITLE
build: dataclasses-avroschema example added. Closes #3

### LINKED ISSUE #3: Add dataclasses-avroschema example
We want to add a [dataclasses-avroschema](https://github.com/marcosschroh/dataclasses-avroschema) example for `custom serialization`

### PR DESCRIPTION
**Pull request by marcosschroh**

### CODE REVIEW

File: examples/dataclasses-avroschema-example/dataclasses_avroschema_example/app.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    value_serializer=serializers.AvroSerializer(),
+)
+
+
+@stream_engine.stream(
+    user_topic, value_deserializer=serializers.AvroDeserializer(model=User)
+)
+async def user_stream(stream: Stream):
+    async for cr in stream:
+        print(f"Event consumed on topic {user_topic}. The user is {cr.value}")
```
Comment (Woile) [inline examples/dataclasses-avroschema-example/dataclasses_avroschema_example/app.py:21]: maybe also print the type of `cr.value`:
```python
print(f"Event consumed on topic {user_topic}. The user is {cr.value}. User type: {type(cr.value)}")
```

Comment (marcosschroh) [reply examples/dataclasses-avroschema-example/dataclasses_avroschema_example/app.py:21]: it is not necessary as `cr.value` is an instance and it has the `__repr__()` method. It is clear in the `logs` to see what is what.

File: examples/dataclasses-avroschema-example/dataclasses_avroschema_example/app.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+@stream_engine.stream(
+    address_topic, value_deserializer=serializers.AvroDeserializer(model=Address)
+)
+async def address_stream(stream: Stream):
+    async for cr in stream:
+        print(f"Event consumed on topic {address_topic}. The Address is {cr.value}")
+
+
+async def produce():
+    for _ in range(5):
```
Comment (Woile) [inline examples/dataclasses-avroschema-example/dataclasses_avroschema_example/app.py:33]: maybe `index` could be used as user id?
```python
for index in range(5):
            await stream_engine.send(
            user_topic,
            value=User.fake(pk=index),
        )
        await stream_engine.send(
            address_topic,
            value=Address.fake(user=index),
        )

        await asyncio.sleep(3)
```

File: examples/dataclasses-avroschema-example/README.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    class Meta:
+        namespace = "Address.v1"
+        aliases = ["address-v1",]
+```
+
+To accomplish it, we need to define custom `serializers` for our `models`:
+
+```python
+# serializers.py
+class AvroSerializer:
```
Comment (Woile) [inline examples/dataclasses-avroschema-example/README.md:56]: Tecnically, you don't event need this, right?

```python
class KstreamSerializer:
     async def serialize(self, instance: AvroModel, **kwargs) -> bytes:
         ...
     async def deserialize(
        self, consumer_record: aiokafka.structs.ConsumerRecord, **kwargs
    ) -> aiokafka.structs.ConsumerRecord:
        ...

class KAvroModel(AvroModel, KstreamSerializer):
    ...

@dataclass
class User(KAvroModel):
     ...


@stream_engine.stream(
    user_topic, value_deserializer=User
)
async def user_stream(stream: Stream):
    async for cr in stream:
        print(f"Event consumed on topic {user_topic}. The user is {cr.value}")
```

Comment (marcosschroh) [reply examples/dataclasses-avroschema-example/README.md:56]: Your approach is incorrect:

1. `deserialize` is a `instance` method and you are using a class to make reference in the `stream`: value_deserializer=User. 
2. You have multiple inheritance and AvroModel defines `serialize` and `deserialize`, which means that `KstreamSerializer.serialize` and `KstreamSerializer.deserialize` are useless.

**Pull request merged**